efinance/common/config.py

```diff
@@ -1,11 +1,24 @@
+import enum
 from pathlib import Path
 
 HERE = Path(__file__).parent
 
 
+class MarketType(enum.Enum):
+    A_stock = "AStock"
+    Hongkong = "HK"
+    London_stock_exchange = "LSE"
+    US_stock = "UsStock"
+    # there may exists more market type
```

Comment:
Could you please add more market types? You can get more  `classify` by `efinance.utils.search_quote` with stock and stock index from different markets.

For example
```python
In [1]: from efinance.utils import search_quote

In [2]: search_quote('000001',count=2)
Out[2]:
[Quote(code='000001', name='平安银行', pinyin='PAYH', id='0000012', jys='6', classify='AStock', market_type='2', security_typeName='深A', security_type='2', mkt_num='0', type_us='6', quote_id='0.000001', unified_code='000001', inner_code='15855238340410'),
 Quote(code='000001', name='上证指数', pinyin='SZZS', id='0000011', jys='1', classify='Index', market_type='1', security_typeName='指数', security_type='5', mkt_num='1', type_us='1', quote_id='1.000001', unified_code='000001', inner_code='38709193299776')]

In [3]: search_quote('NDX',count=2)
Out[3]:
[Quote(code='NDX', name='纳斯达克', pinyin='NSDK', id='NDX_UI', jys='QQZS', classify='UniversalIndex', market_type='_UI', security_typeName='指数', security_type='11', mkt_num='100', type_us='0', quote_id='100.NDX', unified_code='NDX', inner_code='49387883367286'),
 Quote(code='601198', name='东兴证券', pinyin='DXZQ', id='6011981', jys='2', classify='AStock', market_type='1', security_typeName='沪A', security_type='1', mkt_num='1', type_us='2', quote_id='1.601198', unified_code='601198', inner_code='32322054865723')]
```

Reply:
Below lists classify names I have discovered so far:
```python
class MarketType(enum.Enum):
    A_stock = "AStock"  # A股
    A_stock_index = "Index" # A股指数
    B_stock = "BStock"  # B股
    index = "Index" # 沪深京指数
    STAR_market = "23"  # 科创板
    CSI_free_float = "24" # 中证系列指数
    NEEQ = "NEEQ"   # 京A，新三板（全国中小企业股份转让系统）
    BK = "BK"   # 板块
    Hongkong = "HK" # 港股
    US_stock = "UsStock"    # 美股
    London_stock = "LSE"   # 英股
    London_stock_international = "LSEIOB"   # 伦敦交易所国际挂盘册
    universal_index = "UniversalIndex"  # 国外指数
    SIX_Swiss = "SIX" # SIX瑞士股市
```
I think the granularity of market type might be too fined for users. For example, when users want to search for indices in China, they probably expect API to search from both "A_stock_index", "CSI_free_float" and "index", what do you think?